CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to D1GT (Diablo 1 Graphics Tool)
+
+Whenever you want to contribute to the project, whether it's by editing a simple file and fixing a typo, or implementing a huge feature which will add something new to the application - be sure that the change you are making is in a direction that the project is heading. You can make sure by either opening an issue and discussing it here first, or by asking on our discord sever, on an appropriate channel, like `#development`.
+
+## Communication
+
+Join our Discord server: [Diasurgical Discord](https://discord.gg/devilutionx)
+
+## Creating an issue - guidelines
+
+Issues are mostly created during the following occassions:
+
+* There is a weird/undefined behavior of the app, and you're not sure if it's a bug or something that is expected to happen
+* There is an obvious bug that for example causes the app to crash
+* You're not sure if the implementation you want to make aligns with the project's direction - here you can create an issue to propose your implementation change and see if community approves it
+
+For questions regarding your implementation change, you can always ask on our Discord, on `#development` channel.
+
+Please try to do these when creating an issue:
+
+* Try to prepend your issue's title with the category/component that the issue is related to, for example, if your change is mostly related to LevelCelView, you'd name your issue as follows: `LevelCelView: Modified <...>`. If you're not sure, you can drop the tag, and if the issue is most probably related to every, or many components of the app, you can use `Everywhere:` tag.
+* Create only one issue per bug/implementation request
+* Thoroughly describe the bug that you witnessed, if possible also add screenshots - there is no general pattern or template on how the issue should look like, so try to give the next person who will take care of it as many hints as possible
+* Include your platform name that you've witnessed the bug on, and libraries versions that you're using on your system
+* Do not create issues regarding build issues or other support request, if CI builds the application during pull request - the problem is most probably on your side. Please ask for help on `#request-support` channel on our Discord.
```

Comment:
[discussion](https://github.com/diasurgical/d1-graphics-tool/discussions) on github would also be ok

Reply:
Added a reference link to the discussions tab on github, so contributors can redirect build issues here.